js/stanford_events_importer.js

```diff
@@ -231,37 +231,36 @@
 // -----------------------------------------------------------------------------
 
   if(typeof $.fn.quiet_highlight !== "function") {
-      /**
+    /**
      * A quick yellow highlight that fades out.
      */
     $.fn.quiet_highlight = function() {
```

Comment:
There was a reason but it escapes me right now. I think it had to do with me not wanting to add the fade/highlight to the picker module but I suspect I did anyway. I will find out.